### PR TITLE
fix limiting on atom format

### PR DESCRIPTION
**Pull request by sktoiva**
Fixes https://sharetribe.airbrake.io/projects/15448/groups/1347153929703886658

Without format: true, constraints on format do not work so that they would limit access only to the specified one.

### CODE REVIEW

**Comment by ovan** [comment]
Seems to be something relevant reported by cucumber:
```
 No route matches {:format=>nil, :controller=>"atom/listings", :locale=>:en} (ActionView::Template::Error)
```

File: Gemfile.lock [hunk 1]
```diff
@@ -398,7 +398,7 @@ GEM
     ruby-ole (1.2.11.6)
     ruby-prof (0.14.2)
     ruby-progressbar (1.4.2)
-    rubyzip (1.1.6)
+    rubyzip (1.1.7)
```
Comment (rap1ds) [inline Gemfile.lock]: What's this?

Comment (sktoiva) [inline Gemfile.lock]: Dunno, dependency to selenium webdriver.

Comment (rap1ds) [inline Gemfile.lock]: Why is it updated?

Comment (sktoiva) [inline Gemfile.lock]: Check commits. I updated selenium webdriver.

Comment (rap1ds) [inline Gemfile.lock]: Ok, but selenium webdriver is not updated, I guess? But the ffi is, which is probably something related.

**Pull request closed without merging**